src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  margin: 0 auto;
+  width: 100%;
+  background-color: $bg-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  z-index: 1000;
+
+  &__container {
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
+
+  &__logo-link {
+    display: block;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__item {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
```

Comment:
The `margin-left: 20px;` property in `.nav__item` class is applied to all items except the first one. However, it might look better if it was applied to all items, including the first one. Consider changing `&:not(:first-child)` to `&:not(:last-child)`.